spec/features/items/index_spec.rb

```diff
@@ -58,5 +58,120 @@
+        @okay_candy = @meg.items.create(name: "Okay candy", description: "You can eat it!", price: 3, image: "https://images-na.ssl-images-amazon.com/images/I/610ZOacuGPL._SL1050_.jpg", inventory: 100)
+        @bad_paper = @meg.items.create(name: "Bad Paper", description: "Bad for writing on!", price: 20, image: "https://cdn.vertex42.com/WordTemplates/images/printable-lined-paper-wide-ruled.png", inventory: 3)
+        @bad_pencil = @meg.items.create(name: "Bad Pencil", description: "You cannot write on paper with it!", price: 2, image: "https://images-na.ssl-images-amazon.com/images/I/31BlVr01izL._SX425_.jpg", inventory: 100)
+        @bad_candy = @meg.items.create(name: "Bad candy", description: "You cannot eat it!", price: 3, image: "https://images-na.ssl-images-amazon.com/images/I/610ZOacuGPL._SL1050_.jpg", inventory: 100)
+        @very_bad_paper = @meg.items.create(name: "Very Bad Paper", description: "Horrible for writing on!", price: 20, image: "https://cdn.vertex42.com/WordTemplates/images/printable-lined-paper-wide-ruled.png", inventory: 3)
+        @very_bad_pencil = @meg.items.create(name: "Very Bad Pencil", description: "You get hurt by it!", price: 2, image: "https://images-na.ssl-images-amazon.com/images/I/31BlVr01izL._SX425_.jpg", inventory: 100)
+
+        11.times do
+          visit "/items/#{@pull_toy.id}"
+          click_on "Add To Cart"
+        end
+
+        10.times do
+          visit "/items/#{@paper.id}"
+          click_on "Add To Cart"
+        end
+
+        9.times do
+          visit "/items/#{@tire.id}"
+          click_on "Add To Cart"
+        end
+
+        8.times do
+          visit "/items/#{@pencil.id}"
+          click_on "Add To Cart"
+        end
+
+        7.times do
+          visit "/items/#{@candy.id}"
+          click_on "Add To Cart"
+        end
+
+        5.times do
+          visit "/items/#{@bad_paper.id}"
+          click_on "Add To Cart"
+        end
+
+        4.times do
+          visit "/items/#{@bad_pencil.id}"
+          click_on "Add To Cart"
+        end
+
+        3.times do
+          visit "/items/#{@bad_candy.id}"
+          click_on "Add To Cart"
+        end
+
+        2.times do
+          visit "/items/#{@very_bad_paper.id}"
+          click_on "Add To Cart"
+        end
+
+        visit "/items/#{@very_bad_pencil.id}"
+        click_on "Add To Cart"
+
+        visit "/cart"
+        click_on "Checkout"
+
+        name = "Bert"
+        address = "123 Sesame St."
+        city = "NYC"
+        state = "New York"
+        zip = 10001
+
+        fill_in :name, with: name
+        fill_in :address, with: address
+        fill_in :city, with: city
+        fill_in :state, with: state
+        fill_in :zip, with: zip
+
+        click_button "Create Order"
+
+        visit '/items'
+      end
+
+      it "can show 5 most and least popular items and quantity bought" do
```

Comment:
Is this US18? Did this come in on accident?

Reply:
Yeah, this must be a part of that.  I'll note it so when we merge Everything we're all working on into Dev that we make sure there isn't duplication on this.